protobuf/v1/testing.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package mayastor.v1;
+
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+import "common.proto";
+
+service TestingRpc {
+  // Fault injection.
+  rpc AddInjection (AddInjectionRequest) returns (google.protobuf.Empty) {}
+  rpc RemoveInjection (RemoveInjectionRequest) returns (google.protobuf.Empty) {}
+  rpc ListInjections (ListInjectionsRequest) returns (ListInjectionsReply) {}
+}
+
+message Injection {
+  string uri = 1;
+  string device_name = 2;
+  bool is_active = 3;
```

Comment:
What does active mean?

Reply:
It means a condition that triggers this particular fault injection has been met, and error are being injected.